### ISSUE TITLE
Konsole keeps opening itself.

### ISSUE THREAD

**Issue by EYH0602** [microsoft/wslg#64]
# Environment

```none
Windows build number: Microsoft Windows NT 10.0.21364.0
Your Distribution version:20.04
Your WSLg version: 1.0.17.1
```

# Steps to reproduce

Please check the following Feedback Hub link for detail

https://aka.ms/AAc1cw4

**Short Steps to reproduce**
1. Download `Konsole` in wsl
```bash
sudo apt update
sudo apt install konsole -y
```

2.  open `Konsole (Ubuntu-20.04)` in Windows menu (Windows key).

## Steps 

**WSL logs**: 

* Attach WSLg logs from  `/mnt/wslg`

You can access the wslg logs using explorer at: `\\wsl\<Distro-Name>\mnt\wslg` (eg: `\\wsl\Ubuntu-20.04\mnt\wslg`)

* `puseaudio.log`

[pulseaudio.log](https://github.com/microsoft/wslg/files/6365806/pulseaudio.log)

* `weston.log`

[weston.log](https://github.com/microsoft/wslg/files/6365811/weston.log)

* `versions.txt`

```none
WSLg ( x86_64 ): 1.0.17+3.Branch.master.Sha.a526dfd5ad03d126bb2d8c528f6c3563e86a40da                                  
Mariner: VERSION="1.0.20210224"
FreeRDP: e4a2fc2053bd8c5f99455fcd08ffee7e5591567a
weston: fd961f5cd116c9358d82ce94d139c1578e21bd00
pulseaudio: 2f0f0b8c3872780f15e275fc12899f4564f01bd5
mesa:
```

#  Expected behavior

`Konsole` opens one window.

# Actual behavior

`Konsole` keeps opening itself endlessly.

**Comment by crramirez** [comment]
Some clarification here:

1. It only happens using the shortcut in start menu, if you execute it from terminal it will run normally
2. When you have GPU accelerated, all the OpenGL programs say (at least for NVIDIA) something about the GPU. When Konsole is started it prints these two lines before opening itself and print them again.
3. I have not seen this behaviour in other application

Regards

**Comment by hideyukn88** [comment]
@crramirez , thanks for report. I confirmed the behavior launching konsole from start menu, it opens many window, it looks it spawn itself repeatedly as child process. 

For 2, for reference, would you please paste the text here ?

**Comment by hideyukn88** [comment]
This is because when starting program from start menu, it sets the launching program as SHELL, in this case konsole, but konsole read SHELL env and start it as child process assuming it's bash or sh, thus this causes konsole starting itself repeatedly. We will look into the solution for this, thanks for reporting!

**Comment by crramirez** [comment]
Hello @hideyukn88,

This is the message that glxinfo and konsole shows with NVidia driver:

```
NVD3D10: CPU cyclestats are disabled on client virtualization
NVD3D10: CPU cyclestats are disabled on client virtualization
```

To be fair. I didn't report the issue, @EYH0602 was. 

I experienced this problem but I didn't reported because I was using Pengwin, and I didn't know if it was a Pengwin issue or a general one. I was waiting to test in another distro and @EYH0602 reported it for Ubuntu. So, I added my experience.

Regards,
Carlos
